.github/workflows/docs.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     - name: Setup .NET Core
       uses: actions/setup-dotnet@v1
       with:
-        dotnet-version: 5.0.400
```

Comment:
instead of having to keep these versions up to date, if you have a global.json file in this repo you can update these actions to be:

```yaml
- name: Setup .NET Core
  uses: action/setup-dotnet@v2
```

and the action will use the version constraints from the global.json, keeping them in sync from that point onward.

Reply:
Looks like we need two sdks unfortunately, but it's all a bit messy.  FsDocs likes to run with .NET 6 and we're currently building docs from "main" of dotnet/fsahrp, which likes whatever .NET 7 preview it's on.

Reply:
I'm not even totally sure this separate docs build is worth it tbh, but if we want generated docs the only other solution would be to generate within dotnet/fsharp itself.  But that would probably lead to integrating all of FSharp.Formatting into dotnet/fsharp, which is a big hit, leading to more and more monorepo.  Perhaps it's what we should do.